lib/sacastats_web/templates/layout/live.html.heex

```diff
@@ -1,11 +1,13 @@
 <main class="container">
-  <p class="alert alert-info" role="alert"
-    phx-click_x="lv:clear-flash"
-    phx-value-key="info"><%= live_flash(@flash, :info) %></p>
-
-  <p class="alert alert-danger" role="alert"
-    phx-click_x="lv:clear-flash"
-    phx-value-key="error"><%= live_flash(@flash, :error) %></p>
+  <!-- Flash for non character pages as they already have one -->
```

Comment:
If the flashes for character pages are fetched somewhere else, I think those should be removed in favor of these, which will fetch it for all pages

Reply:
Same as above.